### PR TITLE
Update the options' descriptions of the neo4j-admin commands according to the code

### PR DESCRIPTION
**Pull request by NataliaIvakina**
see https://github.com/neo-technology/neo4j/pull/27466, https://github.com/neo-technology/neo4j/pull/27502, and https://github.com/neo-technology/neo4j/pull/27550

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Review by renetapopova — COMMENTED** [review]
It looks good. I am just concerned that we are mentioning the versions in the code. In 2025.01, we'll need to remove them just like the labels. I think a better practice is to keep the code clean and leave all labels to the docs.

File: modules/ROOT/pages/backup-restore/aggregate.adoc [hunk 1]
```diff
@@ -71,7 +71,7 @@ Aggregates a chain of backup artifacts into a single artifact.
 |Accepts either a path to a single artifact file or a folder containing backup artifacts.
 
 When a file is supplied, the _<database>_ parameter should be omitted.
-The option to supply a file is only available from Neo4j 5.2 onwards.
+It is possible to aggregate backup artifacts from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/aggregate.adoc:74]:
```suggestion
The option to supply a file is only available from Neo4j 5.2 onwards.
It is possible to aggregate backup artifacts from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by renetapopova** [comment]
Could you please update the descriptions as per https://github.com/neo-technology/neo4j/pull/27550? I'll then rebase the Parket PR and see if the test there will fail again.

**Comment by NataliaIvakina** [comment]
> Could you please update the descriptions as per [neo-technology/neo4j#27550](https://github.com/neo-technology/neo4j/pull/27550)? I'll then rebase the Parket PR and see if the test there will fail again.

yes, sure. I'll do that

File: modules/ROOT/pages/backup-restore/online-backup.adoc [hunk 2]
```diff
@@ -174,7 +174,7 @@ Note: this is an EXPERIMENTAL option. Consult Neo4j support before use.
 |
 
 |--to-path=<path>
-|Directory to place backup in (required unless `--inspect-path` is used).
+|Directory to place backup in (required unless `--inspect-path` is used). It is possible to back up databases into AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/online-backup.adoc:177]:
```suggestion
|Directory to place backup in (required unless `--inspect-path` is used). It is possible to back up databases into AWS S3 buckets, Google Cloud storage buckets, and Azure using the appropriate URI as the path.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 3]
```diff
@@ -79,6 +79,7 @@ neo4j-admin database restore [-h] [--expand-commands]
 |--from-path=<path>[,<path>...]
 |A single path or a comma-separated list of paths pointing to a backup artifact file.
 An artifact file can be 1) a full backup, in which case it is restored directly or, 2) a differential backup, in which case the command tries first to find in the folder a backup chain ending at that specific differential backup and then restores that chain.
+It is possible to restore backups from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:82]:
```suggestion
It is possible to restore backups from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

File: modules/ROOT/pages/backup-restore/offline-backup.adoc [hunk 4]
```diff
@@ -82,6 +82,7 @@ The `neo4j-admin database dump` command has the following options:
 
 |--to-path=<path>
 |Destination folder of a database dump.
+It is possible to dump databases into AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/offline-backup.adoc:85]:
```suggestion
It is possible to dump databases into AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 5]
```diff
@@ -77,6 +77,7 @@ If `--info` is specified, then the database is not loaded, but information (i.e.
 
 |--from-path=<path>
 |Path to directory containing archive(s).
+It is possible to load databases from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:80]:
```suggestion
It is possible to load databases from AWS S3 buckets, Google Cloud storage buckets, and Azure bucket using the appropriate URI as the path.
```

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 6]
```diff
@@ -240,7 +240,9 @@ Therefore, use it with care, especially with large imports.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
 
-For an example, see <<import-tool-multiple-input-files-regex-example>>.
+It is possible to import files from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:243]:
```suggestion
It is possible to import files from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 7]
```diff
@@ -751,6 +755,8 @@ Therefore, use it with care, especially with large imports.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
 
+It is possible to import files from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:758]:
```suggestion
It is possible to import files from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 8]
```diff
@@ -240,7 +240,9 @@ Therefore, use it with care, especially with large imports.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
 
-For an example, see <<import-tool-multiple-input-files-regex-example>>.
+It is possible to import files from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
+
+For an example, see [import-tool-multiple-input-files-regex-example].
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:245]:
```suggestion
For an example, see <<import-tool-multiple-input-files-regex-example>>.
```

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 9]
```diff
@@ -274,6 +276,8 @@ The value can be a plain number or a byte units string, e.g. `128k`, `1m`.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
 
+It is possible to import files from AWS S3 buckets (from Neo4j 5.19), Google Cloud storage buckets (from Neo4j 5.21), and Azure buckets (from Neo4j 5.24).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:279]:
```suggestion
It is possible to import files from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 10]
```diff
@@ -781,6 +787,8 @@ The value can be a plain number or a byte units string, e.g. `128k`, `1m`.
...
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:790]:
```suggestion
It is possible to import files from AWS S3 buckets, Google Cloud storage buckets, and Azure buckets using the appropriate URI as the path.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1852.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Import](https://neo4j-docs-operations-1852.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Pull request merged**